src/components/layout/Navigation.vue

```diff
@@ -38,9 +38,9 @@
       class="navigation__item profile"
       @click="scrollTop()"
     >
-      <router-link :to="{ name: 'create-profile' }">
+      <router-link :to="{ name: 'install-wallet' }">
         <div class="navigation__item__image" />
-        <span>Create Profile</span>
+        <span>Install Wallet</span>
```

Comment:
May be it should be 'Install a wallet' - it is grammatically correct

Reply:
However the content is kind of a mess there. It says How to Use superhero
So probably we may just rename it Tutorial